tsl/test/expected/dist_ddl.out

```diff
@@ -34,9 +35,6 @@ CALL distributed_exec('CREATE TABLE local(time timestamptz, measure int)', '{ "d
 CALL distributed_exec($$ SELECT create_hypertable('local', 'time') $$, '{ "data_node_1", "data_node_3" }');
 -- Import testsupport.sql file to data nodes
 \unset ECHO
--- This SCHEMA will not be created on data nodes
-CREATE SCHEMA disttable_schema AUTHORIZATION :ROLE_1;
```

Comment:
Why was this removed? Is `disttable_schema` not used?

Reply:
Because it would not work now, the tests bellow assumed that this schema was not created on the data nodes. I've moved schema creation down in the code, where the CREATE SCHEMA is actually tested